### DISCUSSION TITLE
number of approvals to merge branch

### DISCUSSION THREAD

**Discussion by Mohamed-Elnageeb** [MIT-Emerging-Talent/ET6-foundations-group-24#60] [Polls]
hello guys, 
Currently, our repository requires only **one approval** for pull requests (PRs) before merging. This is incosistent with what we agrred upon ( having two reviewers per task) so I want to increase the required approvals for PRs to **two reviewers** in the branch protection rules.

![image](https://github.com/user-attachments/assets/7e0d013a-6805-4647-94d4-ad9d75693f03)

so what do you guys think

**Comment by GaiSamuel** [comment]
Actually, we can leave it as it is because we will be reviewing one task per person, and then others can later add their reviews once they are done with their assigned tasks. But I'm not sure every task will get more than one review.

**Reply by Mohamed-Elnageeb** [reply]
but we assigned to reviewers per task and agreed upon it , this makes it abit inconsistent with the settings

**Reply by GaiSamuel** [reply]
Just noticed now yeah you right and the setting being 1 needed approval might become a problem if someone merges changes when the 2nd person hasn't reviewed

**Comment by Zeinab15** [comment]
So based on this anyone should review 4 challenges right ? @Mohamed-Elnageeb

**Reply by GaiSamuel** [reply]
We are reviewing two challenges of each person we are assigned to. That makes it 4 yeah

**Reply by Mohamed-Elnageeb** [reply]
On average yes
